social_protection/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from core import models as core_models
+from policyholder.models import PolicyHolder
+
+
+class BenefitPlan(core_models.HistoryBusinessModel):
+    code = models.CharField(db_column='Code', max_length=8, null=False)
+    name = models.CharField(db_column='NameBF', max_length=255, null=False)
+    date_from = models.DateTimeField(db_column="DateFrom")
+    date_to = models.DateTimeField(db_column="DateTo")
+    max_beneficiaries = models.SmallIntegerField(db_column="MaxNoBeneficiaries")
+    ceiling_per_beneficiary = models.DecimalField(db_column="BeneficiaryCeiling",
+                                                  max_digits=18,
+                                                  decimal_places=2,
+                                                  blank=True,
+                                                  null=True,
+                                                  )
```

Comment:
For now we can keep this in model directly however we already should think on attaching something like `BenefitPlanPolicy`that would allow us not to rely on fixed fields but something more generic. Please check with Seweryn if calculation rules would be a good use for this case.

Reply:
@sniedzielski Could please take look at it?

Reply:
I think this is a good point to use later here calculation rule however we can do this later when we will know what kind of calculation rule is supposed to be used in that context. (I mean that we don't have any calculation rule related to social protection schemas)

Reply:
@dborowiecki So I am keeping it for now.